api/mailinabox.yml

```diff
@@ -0,0 +1,2219 @@
+    post:
+      tags:
+        - System
+      description: |
+        Retrieve system status. Returns an array of statuses which can include headings.
+      operationId: getSystemStatus
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SystemStatusResponse'
+              example:
+                - type: heading
+                  text: System
+                  extra: []
+                - type: warning
+                  text: This domain's DNSSEC DS record is not set
+                  extra:
+                    - monospace: false
+                      text: 'Digest Type: 2 / SHA-25'
+        403:
+          description: Forbidden
+          content:
+            text/html:
+              schema:
+                type: string
+  /system/version:
+    get:
+      tags:
+        - System
+      description: Retrieve installed Mail-in-a-Box version.
+      operationId: getSystemVersion
+      responses:
+        200:
+          description: Successful operation
+          content:
+            text/html:
+              schema:
+                $ref: '#/components/schemas/SystemVersionResponse'
+        403:
+          description: Forbidden
+          content:
+            text/html:
+              schema:
+                type: string
+  /system/latest-upstream-version:
+    post:
+      tags:
+        - System
+      description: Retrieve Mail-in-a-Box upstream version.
+      operationId: getSystemUpstreamVersion
+      responses:
+        200:
+          description: Successful operation
+          content:
+            text/html:
+              schema:
+                $ref: '#/components/schemas/SystemVersionUpstreamResponse'
+        403:
+          description: Forbidden
+          content:
+            text/html:
+              schema:
+                type: string
+  /system/updates:
+    get:
+      tags:
+        - System
+      description: Retrieve system updates.
+      operationId: getSystemUpdates
+      responses:
+        200:
+          description: Successful operation
+          content:
+            text/html:
+              schema:
+                $ref: '#/components/schemas/SystemUpdatesResponse'
+        403:
+          description: Forbidden
+          content:
+            text/html:
+              schema:
+                type: string
+  /system/update-packages:
+    post:
+      tags:
+        - System
+      description: Update system packages.
+      operationId: updateSystemPackages
+      responses:
+        200:
+          description: Successful operation
+          content:
+            text/html:
+              schema:
+                $ref: '#/components/schemas/SystemUpdatePackagesResponse'
+        403:
+          description: Forbidden
+          content:
+            text/html:
+              schema:
+                type: string
+        5XX:
```

Comment:
While this response code range is [perfectly valid](https://swagger.io/docs/specification/describing-responses/), i've noticed some of the openapi generators don't support it. It might worth changing the approach to use a "whildcard" [default response](https://swagger.io/docs/specification/describing-responses/#default) for unexpected errors.